build_rules.gradle

```diff
@@ -507,3 +507,20 @@ ext.applyAvroNature = {
   println "applyAvroNature with " + (it ? "$it" : "default configuration") + " for project $project.name"
   apply plugin: "com.commercehub.gradle.plugin.avro"
 }
+
+// Creates a task to run the quickstart for a runner.
+// Releases version and URL, can be overriden for a RC release with
+// ./gradlew :release:runQuickstartJava -Pver=2.3.0 -Prepourl=https://repository.apache.org/content/repositories/orgapachebeam-1027
+ext.createJavaQuickstartValidationTask = { String name, extra_args=[] ->
```

Comment:
Take a map instead of positional arguments like applyJavaNature.

Positional arguments make it really difficult in groovy to know what is being called with what and they don't work well with code changes that add/remove default values.

It also allows you to document the configuration objects fields.

Reply:
Done. Created a JavaQuickstartConfiguration similar to applyJavaNature and used map notation in all the calls.